starter_code/hangman.js

```diff
@@ -1,42 +1,47 @@
 var hangman;
 
-// function Hangman() {
+function Hangman() {
+  this.words = ["hello","please","thanks"];
+  this.secretWord = "";
+  this.letters = [];
+  this.guessedLetters = "";
+  this.errorsLeft = 10;
+}
 
-// }
+Hangman.prototype.getWord = function () {
+  return this.words[Math.floor(Math.random()*this.words.length)];
 
-// Hangman.prototype.getWord = function () {
-
-// };
-
-// Hangman.prototype.checkIfLetter = function (keyCode) {
-
-// };
+};
 
-// Hangman.prototype.checkClickedLetters = function (key) {
+Hangman.prototype.checkIfLetter = function (keyCode) {
+    return ((keyCode>64 && keyCode<91) ? true  : false );
+};
 
-// };
+Hangman.prototype.checkClickedLetters = function (key) {
+  return !this.letters.includes(key);
+};
 
-// Hangman.prototype.addCorrectLetter = function (i) {
+Hangman.prototype.addCorrectLetter = function (i) {
+//   return (this.clickedLetters) ? this.guessedLetter += this.secretWord[i].toUpperCase() :
+};
 
-// };
+Hangman.prototype.addWrongLetter = function (letter) {
 
-// Hangman.prototype.addWrongLetter = function (letter) {
+};
 
-// };
+Hangman.prototype.checkGameOver = function () {
```

Comment:
Here you just have to check if the counter is equal to 0, if it is you return true if not return false : 

```js

Hangman.prototype.checkGameOver = function() {
  if (this.errorsLeft > 0) {
    return false;
  }
  return true;
};
```